src/behaviors/hv-alert/index.ts

```diff
@@ -88,6 +91,7 @@ export default {
     // On Android, alerts don't have a default button when unspecified, so we need to set one.
     if (!options.length && Platform.OS === 'android') {
       options.push({
+        // eslint-disable-next-line @typescript-eslint/no-empty-function
```

Comment:
We can define a `noop` function in a utility module somewhere, disable the lint rule onece, and reuse it instead of having to include the lint disable comment everywhere

```
# util.ts
// eslint-disable-next-line @typescript-eslint/no-empty-function
export const noop = () => {};

# ...hv-alert/index.ts
import { noop } from './util';

...

onPress: noop,
```

Reply:
The cleaner way would be to make `onPress` optional IMO instead of making it mandatory and requiring passing a noop.

Reply:
Ah, I didn't realize that was an option. That would indeed be better!